bazel/scripts/run_integ_tests.sh

```diff
@@ -476,6 +487,11 @@ then
     FLAKY_ARGS=( --force-flaky --no-flaky-report "--max-runs=$((RETRY_ATTEMPTS + 1))" "--min-passes=1" )
 fi
 
+echo "#######################################"
+echo "BUILDING REQUESTED TESTS"
+echo "#######################################"
+build_tests
```

Comment:
Perhaps we can remove the printing in `create_test_targets`, because now both `build_tests` and `create_test_targets` print all test targets when you run without options?

Reply:
`create_test_targets` is used for printing for the listing functionality. I think printing the targets individually and then together as a bazel command is okay.